src/core/variables.tf

```diff
@@ -384,13 +384,13 @@ variable "app_backend_plan_sku_size" {
 variable "app_backend_autoscale_minimum" {
   type        = number
   description = "The minimum number of instances for this resource."
-  default     = 1
+  default     = 3
```

Comment:
min 10 and default 10?

Reply:
we are going to spend an arm and a leg :-(

Reply:
will this be a temp setting for cgn launch? or definitive?